community/settings.py

```diff
@@ -11,7 +11,7 @@
 """
 
 import os
-
+from gci.config import get_api_key
```

Comment:
this causes a cyclic import.

you could move the OH_TOKEN constant into the openhub app, which is the only place it is used.

Reply:
Thanks :)